deploy/manifests/base/kong-ingress-dbless.yaml

```diff
@@ -39,6 +39,8 @@ spec:
           value: /dev/stderr
         - name: KONG_ADMIN_LISTEN
           value: 127.0.0.1:8444 ssl
+        - name: KONG_PROXY_LISTEN
+          value: 0.0.0.0:8000, 0.0.0.0:9080 http2, 0.0.0.0:8443 ssl http2
```

Comment:
As we discussed offline, we should enable http2 on the `8443` port but we don't need to enable plain-text version of `http2` by default. Users can enable it if they wish to.
I don't know if there is widespread adoption of http2 in plain-text because of backwards incompatibility with http1.

So, let's remove `9080`.